.github/workflows/pipelines-delegated.yml

```diff
@@ -8,20 +8,34 @@ on:
 
 env:
   PIPELINES_CLI_VERSION: v0.9.8
+  PIPELINES_ACTIONS_VERSION: main
```

Comment:
Do we want to pin the actions repo instead of main?

Reply:
Pinned to v1.0.1